src/app/handlers/command.handler.ts

```diff
@@ -69,6 +70,12 @@ export class CommandHandler implements types.IHandler {
     await msg.react(this._CHECK_EMOTE);
     await msg.react(this._CANCEL_EMOTE);
 
+    // Delete Message after timeout
+    // Delete the message after given timeout has passed.
+    setTimeout(() => {
+      msg.delete();
+    }, 1000 * CommandHandler._EXPIRE_SECONDS);
+
```

Comment:
Since there is a collector on this message, it would make more sense to delete after the collector ends
so you could do
```
collector.on('end', () => msg.delete())
```